src/4.3_to_5.0/migrate_43_to_50.ts

```diff
@@ -458,6 +445,25 @@ export async function migrate_43_to_50(
               name: bookmark.name,
               step: "4.3 to 5.0",
             });
+            if (error instanceof PartialMigrationError) {
+              counters.widgets.partial++;
+              migratedWidget = error.migratedWidgetState;
+            } else {
+              counters.widgets.failed++;
+              // The migration failed.
+              // The widget is kept as is.
+              migratedWidget = bookmark;
+              widgets[fileId] = migratedWidget;
```

Comment:
Why do we need to do this here? Isn't it already taken care of in https://github.com/activeviam/atoti-ui-migration/blob/c88337116110b275c174540fe940b69d0bfcec9d/src/4.3_to_5.0/migrate_43_to_50.ts#L464?

Reply:
@Nouzbe I undid my changes within `migrate_4.3_to_5.0` as indeed, preserving the bookmarks content was handled by this. There was a slight modification to one of the tests but all tests should still pass. This makes this PR more simple 👍